twisted/internet/defer.py

```diff
@@ -161,7 +166,7 @@ def maybeDeferred(f, *args, **kw):
 
 
 def timeout(deferred):
-    deferred.errback(failure.Failure(TimeoutError("Callback timed out")))
+    deferred.errback(failure.Failure(error.TimeoutError("Callback timed out")))
```

Comment:
True. Let's deprecate it and the deprecation test should handle the coverage:)

It can be deprecated in this branch as it is somehow related to defer.TimeoutError